package.json

```diff
@@ -25,14 +25,23 @@
     "gatsby-plugin"
   ],
   "dependencies": {
+    "remark": "^13.0.0",
```

Comment:
I don't think we need `remark` as a dependency, could instead be a `devDependency`.